src/System.ComponentModel.Primitives/ref/project.json

```diff
@@ -1,13 +1,31 @@
 {
-  "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.ComponentModel": "4.0.0"
-  },
   "frameworks": {
-    "netstandard1.0": {
-      "imports": [
-        "dotnet5.1"
-      ]
+    "netstandard1.7": {
+        "dependencies":
+        {
+            "System.Runtime": "4.3.0-beta-24522-03",
```

Comment:
I really hope System.ComponentModel.Primitives doesn't actually depend on all these packages. Certainly the ref doesn't. Please trim this to the exact set of references that show up in the assembly reference table of the built binary.